tests/toolchain/install.pm

```diff
@@ -0,0 +1,46 @@
+use base "opensusebasetest";
+use strict;
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    send_key 'ctrl-alt-f4';
+    assert_screen 'tty4-selected';
+    assert_screen 'text-login';
+    type_string "root\n";
+    assert_screen 'password-prompt', 10;
+    type_password;
+    send_key "ret";
+    assert_screen 'text-logged-in';
+    # disable packagekitd
+    type_string "chmod 444 /usr/lib/packagekitd\n"; # packagekitd will be not executable
+    type_string "killall packagekitd\n";
+    # use repo due to missing libraries http://10.100.98.90/tests/1425/modules/install/steps/5
+    type_string "zypper ar http://download.suse.de/ibs/SUSE:/Maintenance:/783/SUSE_Updates_SLE-SERVER_12_x86_64/SUSE:Maintenance:783.repo\n";
+    sleep 5;
+    type_string "zypper -n in gcc5 gcc5-c++ binutils gdb && echo 'PASSED'|tee /dev/$serialdev\n";
+    wait_serial("PASSED", 100);
+    type_string "zypper -n up && echo 'OK'\n";
+    type_string "zypper -n patch && echo 'OK'\n";
```

Comment:
if you're doing an `zypper up`, I'm pretty sure that will automatically include everything for the `zypper patch`.. I think..I'm not sure.